recipes/peppy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "peppy" %}
+{% set version = "0.30.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 058703fd51eeb5dfb57f0dd2bc24ff11c7d999773f43787d76628138697992aa
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
It looks like there is a python entry point for this project:

https://github.com/pepkit/peppy/blob/master/setup.py#L90

See:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and add tests for the cli using:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands